BBGTC/food-bank-app
# feat(expo): Add expo-jest and email validator util

# Expo testing enabled

I enabled Jest unit tests for the frontend.

## How to run tests
1. `cd` into the frontend directory.
2. Run `npm run test` and you're good.

Make sure you run tests regularly before sending a meaningful PR.
